modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/PrimaryRecipients.tsx

```diff
@@ -91,7 +185,7 @@ export function PrimaryRecipient({
 				<ClayTooltipProvider>
 					<span
 						title={Liferay.Language.get(
-							'each-to-recipient-will-receive-separate-emails'
+							'each-to-primaryRecipient-will-receive-separate-emails'
```

Comment:
Hey @larissacribeiro, could you confirm if this change was intencional?

Reply:
it was not, thanks!